src/main/java/com/github/s4u/plugins/PGPVerifyMojo.java

```diff
@@ -136,14 +136,6 @@
     private boolean failWeakSignature;
 
     /**
-     * Verify pom files also.
-     *
-     * @since 1.1.0
-     */
-    @Parameter(property = "pgpverify.verifyPomFiles", defaultValue = "true")
-    private boolean verifyPomFiles;
```

Comment:
Maybe better will be left this property for one release because this breakes compatibility.
We can remove it in next release.